templates.go

```diff
@@ -13,15 +13,18 @@ type Template struct {
 	Format                string            `json:"format,omitempty" doc:"the format of the template."`
 	HostID                *UUID             `json:"hostid,omitempty" doc:"the ID of the secondary storage host for the template"`
 	HostName              string            `json:"hostname,omitempty" doc:"the name of the secondary storage host for the template"`
+	Hypervisor            string            `json:"hypervisor,omitempty" doc:"the target hypervisor for the template"`
```

Comment:
wasn't this root only?

Reply:
Nope, this field is also visible for non-root users.